packages/kit/src/exports/public.d.ts

```diff
@@ -35,6 +35,16 @@ export interface Adapter {
 	 * @param builder An object provided by SvelteKit that contains methods for adapting the app
 	 */
 	adapt(builder: Builder): MaybePromise<void>;
+	/**
+	 * Functions called during dev and build to determine whether specific features will work in production with this adapter
```

Comment:
I don't know if it's premature, but I imagine not everything would be a function in the future and you could also just have some simple booleans for some things. E.g. You might have `ssr: false` for `adapter-static`
```suggestion
	 * Checks called during dev and build to determine whether specific features will work in production with this adapter
```